src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -486,29 +485,39 @@ private void enqueueAndHandleVariant(final VariantContext variant, final Referen
             }
         }
 
-        // Create a place to keep our funcotations:
-        final List<Funcotation> funcotations = new ArrayList<>();
+        // Create only the gencode funcotations.
+        if (retriveGencodeFuncotationFactoryStream().count() > 1) {
```

Comment:
Strictly speaking, this isn't the case anymore - you can have as many `Gencode` data sources as you want, so this may need to be changed.